### PR TITLE
Fix problem setting environment on Logster config

### PR DESCRIPTION
**Pull request by koss-lebedev**
Hey, this resolves a small issue described in #37

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea/
```
Comment (tgxworld) [inline .gitignore]: Can we remove this since this is not related to the issue?

**Comment by tgxworld** [comment]
Fix looks good to me :+1:

File: .gitignore (hunk 1 above)
Comment (koss-lebedev) [inline .gitignore]: Yes, sure. Should I create a new pull request for it?

**Comment by koss-lebedev** [comment]
@tgxworld I removed that unrelated line, can you now merge it?

**Comment by tgxworld** [comment]
Thanks @koss-lebedev  👍
